packages/lodestar/test/memory/testRunnerMemory.ts

```diff
@@ -0,0 +1,215 @@
+export type TestRunnerMemoryOpts<T> = {
```

Comment:
When you're ready, maybe this should be pulled into its own package. I see you're using it in ssz too [here](https://github.com/ChainSafe/ssz/commit/4245cb6c79602e773a17ee7c1c4bad2fec66f35a)

Reply:
Agree it should eventually, tho it still needs more testing.